backpack-compose/src/main/kotlin/net/skyscanner/backpack/compose/button/BpkButton.kt

```diff
@@ -0,0 +1,93 @@
+package net.skyscanner.backpack.compose.button
+
+import androidx.compose.foundation.interaction.MutableInteractionSource
+import androidx.compose.foundation.layout.requiredWidth
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.remember
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.painter.Painter
+import net.skyscanner.backpack.compose.button.internal.BpkButtonImpl
+import net.skyscanner.backpack.compose.button.internal.ButtonIcon
+import net.skyscanner.backpack.compose.button.internal.ButtonText
+
+@Composable
+fun BpkButton(
+  text: String,
+  modifier: Modifier = Modifier,
+  size: BpkButtonSize = BpkButtonSize.Default,
+  colors: BpkButtonColors = BpkButtonColors.Primary,
```

Comment:
naming nitpick: as a backpack user i would expect this to still be type - it is what we internally refer to the different options and while it mainly refers to colors there might be other differences. for example specifically `link` should not have horizontal padding to align it with other text (same as the old version)